packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -176,15 +212,39 @@ export class AppBridgeTheme {
         }
     }
 
-    public async createCoverPage(coverPage: CoverPageCreate) {
+    public async createDocumentCategory(category: Omit<DocumentCategoryRequest, 'id'>) {
+        try {
+            return createDocumentCategory(category as DocumentCategory);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async updateDocumentCategory(category: RequireAtLeastOne<DocumentCategoryRequest, 'title' | 'documentId'>) {
+        try {
+            return updateDocumentCategory(category as DocumentCategory);
+        } catch (error) {
+            throw console.error('Error: ', error);
+        }
+    }
+
+    public async deleteDocumentCategory(id: number) {
```

Comment:
```suggestion
    public async deleteDocumentCategoryById(id: number) {
```
And could you do it for all other `delete[...]`

For some models, we can delete by slug, by foreign id, ...